ikenna-portfolio/portfolio/src/main/java/com/google/sps/servlets/CountServlet.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Metadata;
+import com.google.sps.data.Metadata.Search;
+import com.google.sps.database.CommentDatabase;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.*; 
+
+/** Servlet that returns JSON content. */
+@WebServlet("/count")
+public class CountServlet extends HttpServlet {
+
+  private int count;
```

Comment:
Servlet should be stateless. Consider remove these fields.

Reply:
done